### PR TITLE
Remove iast-log and review ASM log messages

### PR DESCRIPTION
**Pull request by iunanua**
### What does this PR do?
- Remove iast-log
- Include a message in each log.error including the `[ASM]` prefix

### Motivation
With `log` sending error via telemetry, `iast-log` is not necessary

### Plugin Checklist

- [ ] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/library_config/nodejs.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

File: packages/dd-trace/src/appsec/iast/iast-plugin.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class IastPlugin extends Plugin {
       try {
         handler(message, name)
       } catch (e) {
-        iastLog.errorAndPublish(e)
+        log.error('[ASM] Error executing handler', e)
```
Comment (uurien) [inline packages/dd-trace/src/appsec/iast/iast-plugin.js:68]: To try to add more information in the message.

```suggestion
        log.error('[ASM] Error executing IAST plugin handler', e)
```

File: packages/dd-trace/src/appsec/rasp/fs-plugin.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ function enable (mod) {
     fsPlugin.enable()
   }
 
-  log.info(`Enabled AppsecFsPlugin for ${mod}`)
+  log.info(`[ASM] Enabled AppsecFsPlugin for ${mod}`)
```
Comment (CarlesDD) [inline packages/dd-trace/src/appsec/rasp/fs-plugin.js:73]:
```suggestion
  log.info('[ASM] Enabled AppsecFsPlugin for %s', mod)
```

File: packages/dd-trace/src/appsec/rasp/fs-plugin.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ function disable (mod) {
     fsPlugin = undefined
   }
 
-  log.info(`Disabled AppsecFsPlugin for ${mod}`)
+  log.info(`[ASM] Disabled AppsecFsPlugin for ${mod}`)
```
Comment (CarlesDD) [inline packages/dd-trace/src/appsec/rasp/fs-plugin.js:88]:
```suggestion
  log.info('[ASM] Enabled AppsecFsPlugin for %s', mod)
```

Comment (iunanua) [reply packages/dd-trace/src/appsec/rasp/fs-plugin.js]: changed

**Review by CarlesDD — APPROVED** [review]

**Review by uurien — APPROVED** [review]

**Pull request merged**